platform/vs/docker-sonic-vs/supervisord.conf.j2

```diff
@@ -42,6 +42,9 @@ autostart=false
 autorestart=false
 stdout_logfile=syslog
 stderr_logfile=syslog
+{% if ENABLE_ASAN == "y" %}
+environment=ASAN_OPTIONS="log_path=/var/log/asan/syncd-asan.log"
```

Comment:
Does `/var/log/asan/` need to be created first? When testing ASAN for another application, with this `log_path`, ASAN didn't write the log file because `/var/log/asan` didn't exist.

Reply:
For the vs, it is not convenient to have the logs inside of a container. The idea is to map this dir to the host (e.g. -v /asan_reports:/var/log/asan). It makes the automation/analysis of the reports much easier.